assets/wizards/newspack/views/settings/connections/webhooks/endpoint-actions-card.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Settings Wizard: Connections > Webhooks > Endpoint Actions Card
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { getEndpointTitle } from './utils';
+import EndpointActions from './endpoint-actions';
+import WizardsActionCard from '../../../../../wizards-action-card';
+
+function EndpointActionsCard( {
+	endpoint,
+	setAction,
+}: {
+	endpoint: Endpoint;
+	setAction: ( action: WebhookActions, id: number | string ) => void;
+} ) {
+	return (
+		<WizardsActionCard
+			isMedium
+			className="newspack-webhooks__endpoint mt16"
+			toggleChecked={ ! endpoint.disabled }
+			toggleOnChange={ () => setAction( 'toggle', endpoint.id ) }
+			key={ endpoint.id }
+			title={ getEndpointTitle( endpoint ) }
+			disabled={ endpoint.system }
+			description={ () => {
+				if ( endpoint.disabled && endpoint.disabled_error ) {
+					return `${ __(
+						'This endpoint is disabled due to excessive request errors',
```

Comment:
non-blocking question: Are there other reasons the endpoint might be disabled? If so, let's make this message more generic and not specify the reason.

Reply:
Unable to find anything that pertains specifically to Wizards. However, I found that there is some logic in Webhooks internally: https://github.com/Automattic/newspack-plugin/blob/5f34145d37437e41fc260884df7eac9ca90e42cd/includes/data-events/class-webhooks.php#L360-L369 which attempts to recover from errors (x 15) by gradually increasing the delay between attempts seen here - https://github.com/Automattic/newspack-plugin/blob/5f34145d37437e41fc260884df7eac9ca90e42cd/includes/data-events/class-webhooks.php#L752-L768 based on this I've decided to make the error message more generic, as you've recommended.